base/Makefile

```diff
@@ -269,9 +269,11 @@ $(eval $(call symlink_system_library,LIBSUITESPARSE,libamd))
 $(eval $(call symlink_system_library,LIBSUITESPARSE,libcamd))
 $(eval $(call symlink_system_library,LIBSUITESPARSE,libccolamd))
 $(eval $(call symlink_system_library,LIBSUITESPARSE,libcholmod))
+$(eval $(call symlink_system_library,LIBSUITESPARSE,libcholmod_cuda))
```

Comment:
@ViralBShah Are these lines an oversight? `libcholmod_cuda` isn't used anywhere in the PR, and SuiteSparse is built with `-DENABLE_CUDA=0` so they would fail when `USE_SYSTEM_SUITESPARSE=1` if the system SuiteSparse was built using the same settings as Julia's.

Reply:
IIRC, this library was built as a shim, and we had to deliberately add it. See https://github.com/DrTimothyAldenDavis/SuiteSparse/issues/285. 

We should be able to delete it on master.

Reply:
Thanks for the pointer. Actually was just done by #52577.